From 255ed09cbf1f95c86d9c6c53898a7fe8d7e3fd13 Mon Sep 17 00:00:00 2001
From: Arnd Bergmann <arnd@arndb.de>
Date: Tue, 6 Nov 2012 22:49:02 +0100
Subject: [PATCH 498/951] arm-soc: update contents file

After moving the vexpress/soc branch into next/soc2, I updated the
contents file but forgot to push the change. This fixes the file.

Signed-off-by: Arnd Bergmann <arnd@arndb.de>
---
 arch/arm/arm-soc-for-next-contents.txt | 11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

diff --git a/arch/arm/arm-soc-for-next-contents.txt b/arch/arm/arm-soc-for-next-contents.txt
index 3730811..7de5c48 100644
--- a/arch/arm/arm-soc-for-next-contents.txt
+++ b/arch/arm/arm-soc-for-next-contents.txt
@@ -15,7 +15,7 @@ imx/multiplatform	git://git.linaro.org/people/shawnguo/linux-2.6.git imx/multi-p
 			(pulled twice)
 vt8500/cleanup		git://server.prisktech.co.nz/git/linuxwmt.git/ tags/cleanup-for-3.8
 + cherry-pick of "ARM: pxa: use module_platform_driver macro"
-zynq/cleanup		git://git.xilinx.com/linux-xlnx.git arm-next                    
+zynq/cleanup		git://git.xilinx.com/linux-xlnx.git arm-next
 
 next/headers:
 omap/headers		git://git.kernel.org/pub/scm/linux/kernel/git/tmlind/linux-omap omap-for-v3.8/cleanup-headers-signed
@@ -32,7 +32,6 @@ nomadik/plat-removal	git://git.kernel.org/pub/scm/linux/kernel/git/linusw/linux-
 next/soc:
 ux500/core		git://git.kernel.org/pub/scm/linux/kernel/git/linusw/linux-stericsson.git ux500-core
 clps711x/soc		patches from Alexander Shiyan <shc_work@mail.ru>
-vexpress/soc		git://git.linaro.org/people/pawelmoll/linux vexpress-soc
 renesas/soc		git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas soc
 calxeda/ecx-2000	git://sources.calxeda.com/kernel/linux.git tags/calxeda-ecx-2000
 integrator/irq		git://git.kernel.org/pub/scm/linux/kernel/git/linusw/linux-integrator.git tags/integrator-irq
@@ -53,3 +52,11 @@ socfpga/smp		patch from Dinh Nguyen <dinguyen@altera.com>
 next/drivers:
 vexpress/drivers	git://git.linaro.org/people/pawelmoll/linux vexpress/drivers
 
+
+depends/clk:		common clock tree
+
+next/soc2
+vexpress/soc		git://git.linaro.org/people/pawelmoll/linux vexpress-soc
+				depends on depends/clk
+
+
-- 
1.8.0

